### ISSUE TITLE
master user was setup as only a user!

### ISSUE THREAD

**Issue by padders** [userfrosting/UserFrosting#468]
I've just installed userfrosting, but the master user account was only created as a standard user! I had to manually update the uf_group_user table to update the group_id from 1 to 2. Did something go wrong on install? Am I missing something and that's the way it was meant to be setup?

I have had issues with my antivirus preventing jQuery from running properly on the site and when it was initially created it did prevent the page from running properly.

**Comment by padders** [comment]
The user also has access to none of the pages in the default app, other than those defined in the DB which seems limited. I'm rather confused!

**Comment by alexweissman** [comment]
You can see here: https://github.com/userfrosting/UserFrosting/blob/master/userfrosting/controllers/InstallController.php#L235-L241 that the master account is only added to the default groups.

But it actually shouldn't matter, since the master account automatically has access to everything.  So really, it doesn't matter which groups the master account is in.

**Comment by alexweissman** [comment]
I'm also not sure what you mean by 
> The user also has access to none of the pages in the default app, other than those defined in the DB which seems limited.

There shouldn't be any pages defined in the DB.  Are you using UF v0.3.1?

**Comment by padders** [comment]
Yes, I'm using the latest version, v0.3.1. 

I meant the table uf_authorize_group which defines user access to certain functions (unless I've got that wrong too, which is fairly likely!)

The master account only seems to have access to the dashboard default page, and initially it's own user account settings page (from the dropdown in the top right). I updated the tables uf_group_user and uf_user to put it in the admin group, and now it can see the users page too (from the sidebar), but it can no longer access it's own user account page (I get a 404 which I think is down to access)! 

On initial install it also never took me to the site configuration page, and I don't seem to be able to get to that page either. I could be going to an incorrect URL (I'm manually going to www.sitename.com/config/settings).

The other thing I've noticed is when going through the tutorials to create the http://www.sitenamel.com/account/groups page which displays the users groups, that page works fine until I setup the user access permissions. That's as you'd expect, but I can't get to the Configuration->Groups page with the master user as I have no menu option for that, and the master user cannot access the account/groups page any more when I suspect it should be able to get to everything?

I'm fairly sure that either I'm doing something pretty dumb, or something went wrong on install. If I had to bet I'd go for me doing something stupid.
